scripts/flow/flowListener.js

```diff
@@ -83,30 +83,66 @@ const fallbackActionIdentifier_BasicAttackFlow = flow => {
     return flow.state.data.attack_type === "Melee" ? "lwfx_default_melee" : "lwfx_default_ranged";
 };
 
+/**
+ * Convert a named structure flow to a `lwfx` fake LID.
+ *
+ * Includes support for "Lancer Alternative Structure"/"Maria's Alternate Ruleset" rules. This renames some results.
+ * See:
+ *  - https://foundryvtt.com/packages/lancer-alt-structure
+ *  - https://docs.google.com/document/d/1unN3HDDeAK3pN1rmgFgZgAXp5flnQ9-KMu-TXt34tnU/edit
+ */
 const fallbackActionIdentifier_StructureFlow = flow => {
     switch (flow.state.data.title) {
         case "Crushing Hit":
+        case game.i18n.localize("LANCER-ALT-STRUCTURE.StructureTitles.crushingHit"):
```

Comment:
If the module is not enabled, these resolve to the their IDs:

![image](https://github.com/user-attachments/assets/0a2cbc65-c1a0-45ca-ab53-e40e893afff5)

and are skipped over.